features/step_definitions/search_functionality_steps.rb

```diff
@@ -0,0 +1,23 @@
+Given(/^that i am on the landing page$/) do
+  visit root_path
+end
+
+Given(/^there is a field "([^"]*)"$/) do |field|
+  find_field(field).value
```

Comment:
This one should be something along the line of 
```ruby
expect(page).to have_field field
```

But as you mentioned earlier in the call @faraznaeem, this step is useless because we can't do any of the other steps if we have no search field. So you can remove this step